resources/github-actions.md

```diff
@@ -106,13 +106,16 @@ Additional specifications:
 
 The GPU labeled runners are backed by lab machines and have the GPUs specified in the table below installed.
 
-**IMPORTANT**: GPU jobs have two requirements: 1) They **must** run in a container (i.e. `nvidia/cuda:11.5.0-base-ubuntu18.04`) and 2) They must set the {% raw %}`NVIDIA_VISIBLE_DEVICES: ${{ env.NVIDIA_VISIBLE_DEVICES }}`{% endraw %} container environment variable. If these requirements aren't met, the GitHub Actions job will fail. See the _Usage_ section below for an example.
+**IMPORTANT**: GPU jobs have two requirements. If these requirements aren't met, the GitHub Actions job will fail. See the _Usage_ section below for an example.
 
-| Label Combination                | GPU    | Driver Version | # of GPUs |
-| -------------------------------- | ------ | -------------- | --------- |
-| `[linux, amd64, gpu-v100-450-1]` | `V100` | `450`          | `1`       |
-| `[linux, amd64, gpu-v100-495-1]` | `V100` | `495`          | `1`       |
-| `[linux, arm64, gpu-a100-495-1]` | `A100` | `495`          | `1`       |
+1. They **must** run in a container (i.e. `nvidia/cuda:11.5.0-base-ubuntu18.04`)
```

Comment:
1. Use a newer container suggestion (so it'll be up-to-date for longer).
2. No need for the bold.
```suggestion
1. They must run in a container (i.e. `nvidia/cuda:11.8.0-base-ubuntu22.04`)
```